### PR TITLE
port stellar and birte kehr

### PR DESCRIPTION
**Pull request by simonsasse**

### CODE REVIEW

File: _people/birte_kehr.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+---
+name: Birte Kehr
+category: Contributor
+avatar: /assets/images/people/birte_kehr.jpg
+social_links:
+  email: birte.kehr@decode.is
```
Comment (eseiler) [inline _people/birte_kehr.md:6]:
```suggestion
  email: birte.kehr@bihealth.de
```

File: _people/birte_kehr.md [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+  linkedin: birte-kehr-74813175
+---
+
+Dr. Birte Kehr works at the Berlin Institute of Health leading the Junior Research Group Genome Informatics.
```
Comment (eseiler) [inline _people/birte_kehr.md:10]: You can also add https://kehrlab.github.io/ if you want to

Comment (simonsasse) [reply _people/birte_kehr.md:11]: For adding a website I'd have to edit more code, so I just added  github.com/kehrlab/ 
Is that ok ? :)

Comment (eseiler) [reply _people/birte_kehr.md:11]: This is also a good idea.
I had 
```suggestion
Dr. Birte Kehr works at the Berlin Institute of Health leading the [Junior Research Group Genome Informatics](https://kehrlab.github.io/).
```
(+ line break) in mind

File: _people/birte_kehr.md [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+name: Birte Kehr
+category: Contributor
+avatar: /assets/images/people/birte_kehr.jpg
+social_links:
+  email: birte.kehr@bihealth.de
+  linkedin: birte-kehr-74813175
+  github: kehrlab
+---
+
+Dr. Birte Kehr works at the Berlin Institute of Health leading the [Junior Research Group Genome Informatics](https://kehrlab.github.io/).
```
Comment (eseiler) [inline _people/birte_kehr.md:11]:
```suggestion
Dr. Birte Kehr works at the Berlin Institute of Health leading the 
[Junior Research Group Genome Informatics](https://kehrlab.github.io/).
```

**Review by eseiler — APPROVED** [review]

**Pull request merged**